manifest-reference.md

```diff
@@ -353,6 +355,26 @@ In this case the order of the libraries matters:
 link = ["blas", "lapack"]
 ```
 
+## Use system-installed modules
+
+To use modules that are not defined within your fpm package or its dependencies,
+specify the module name using the *external-modules* key in the *build* table.
+
+*Example:*
+
+```toml
+[build]
+external-modules = "netcdf"
+```
+
+Multiple external modules can be specified as a list.
+
+*Example:*
+
+```toml
+[build]
+external-modules = ["netcdff","h5lt"]
```

Comment:
Oops I missed this one.

```suggestion
external-modules = ["netcdf", "h5lt"]
```

Reply:
Should note here that the user is responsible for making the external module available to the compiler in the include path, using a flag or include-dir? As it stands now, it sounds like you only have to specify `external-modules` and you're good to go.

Reply:
Good point, I'll add some text for this.